packages/chain-adapters/src/evm/EvmBaseAdapter.ts

```diff
@@ -303,7 +303,7 @@ export abstract class EvmBaseAdapter<T extends EvmChainId> implements IChainAdap
         value: toHex(isTokenSend ? 0n : BigInt(value)),
         to: isTokenSend ? contractAddress : to,
         chainId: Number(fromChainId(this.chainId).chainReference),
-        data: data || (await getErc20Data(to, value, contractAddress)),
+        data: data || (await getErc20Data(to, value, contractAddress)) || '0x',
```

Comment:
Ensures `0x` is present in case of lack of data (i.e native sends) - SAFE expects it, and other wallets may too